langserve/server.py

```diff
@@ -866,6 +883,46 @@ async def playground(file_path: str, config_hash: str = "") -> Any:
             file_path,
         )
 
+    if enable_feedback_endpoint:
```

Comment:
One decision I went back and forth on but I decided to go with:

We could flag off this endpoint if LangSmith is disabled. However, I kinda like that if the user doesn't opt out, we let this endpoint throw 400s with an error message mentioning langsmith, which increases the visibility of the value of integrating with langsmith